src/openforms/registrations/contrib/stuf_zds/tests/test_backend.py

```diff
@@ -1243,6 +1244,172 @@ def test_plugin_vestiging_initiator_kvk_and_vestigingsnummer(self, m, mock_task)
+        )
+
+        m.post(
+            self.service.soap_service.url,
+            content=load_mock("voegZaakdocumentToe.xml"),
+            additional_matcher=match_text("edcLk01"),
+        )
+        mock_task.id = 1
+
+        form_options = {
+            "zds_zaaktype_code": "zt-code",
+            "zds_zaaktype_omschrijving": "zt-omschrijving",
+            "zds_zaaktype_status_code": "123",
+            "zds_zaaktype_status_omschrijving": "aaabbc",
+            "zds_documenttype_omschrijving_inzending": "aaabbc",
+            # we are deliberately NOT including this to simulate upgrades from earlier
+            # versions where this configuration parameter was not available yet and is
+            # thus missing from the JSON data
+            # "zds_zaakdoc_vertrouwelijkheid": "OPENBAAR",
+        }
+
+        plugin = StufZDSRegistration("stuf")
+        serializer = plugin.configuration_options(data=form_options)
+        self.assertTrue(serializer.is_valid())
+
+        result = plugin.register_submission(submission, serializer.validated_data)
+        self.assertEqual(
+            result,
+            {
+                "zaak": "foo-zaak",
+                "document": "bar-document",
+            },
+        )
+
+        xml_doc = xml_from_request_history(m, 0)
+        self.assertSoapXMLCommon(xml_doc)
+
+        xml_doc = xml_from_request_history(m, 1)
+        self.assertSoapXMLCommon(xml_doc)
+        self.assertXPathEqualDict(
+            xml_doc,
+            {
+                "//zkn:stuurgegevens/stuf:berichtcode": "Lk01",
+                "//zkn:stuurgegevens/stuf:entiteittype": "ZAK",
+                "//zkn:object/zkn:identificatie": "foo-zaak",
+                "//zkn:object/zkn:omschrijving": "my-form",
+                "//zkn:object/zkn:betalingsIndicatie": "N.v.t.",
+                "//zkn:object/zkn:isVan/zkn:gerelateerde/zkn:code": "zt-code",
+                "//zkn:object/zkn:isVan/zkn:gerelateerde/zkn:omschrijving": "zt-omschrijving",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:inp.bsn": "111222333",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:voornamen": "Foo",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:geslachtsnaam": "Bar",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:voorvoegselGeslachtsnaam": "de",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:geboortedatum": "20001231",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:voorletters": "J.W.",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:geslachtsaanduiding": "M",
+                "//zkn:object/zkn:heeftAlsInitiator/zkn:gerelateerde/zkn:medewerker/zkn:identificatie": "123456782",  # Identificatie of the employee
```

Comment:
I don't think that `heeftAlsInitiator` is the correct path here - from what I can gather it needs to go into `//zkn:object/zkn:heeftAlsOverigBetrokkene` with the following schema:

```xml
    <complexType name="ZAKBTROVR-basis">
		<annotation>
			<appinfo>
				<historieMaterieel>J</historieMaterieel>
				<historieFormeel>J</historieFormeel>
			</appinfo>
			<documentation>Relatie naar een overig betrokkene bij de zaak</documentation>
		</annotation>
		<complexContent>
			<restriction base="ZKN:ZAKBTR-basis">
				<sequence>
					<element name="gerelateerde" type="ZKN:BTR-basis" nillable="true" minOccurs="0"/>
					<element name="code" type="ZKN:Code-e" nillable="true" minOccurs="0"/>
					<element name="omschrijving" type="ZKN:Omschrijving-e" nillable="true" minOccurs="0"/>
					<element name="toelichting" type="ZKN:Toelichting-e" nillable="true" minOccurs="0"/>
					<choice minOccurs="0" maxOccurs="2">
						<element name="afwijkendCorrespondentieAdres" type="BG:CorrespondentieAdresGrp-basis" nillable="true" minOccurs="0"/>
						<element name="afwijkendBuitenlandsCorrespondentieAdres" type="BG:VerblijfBuitenlandGrp" nillable="true" minOccurs="0"/>
					</choice>
					<element ref="StUF:tijdvakRelatie" minOccurs="0"/>
					<element ref="StUF:tijdvakGeldigheid" minOccurs="0"/>
					<element ref="StUF:tijdstipRegistratie" minOccurs="0"/>
					<element ref="StUF:extraElementen" minOccurs="0"/>
					<element name="historieMaterieel" type="ZKN:ZAKBTROVR-basis" minOccurs="0" maxOccurs="unbounded"/>
					<element name="historieFormeel" type="ZKN:ZAKBTROVR-basis" minOccurs="0" maxOccurs="unbounded"/>
					<element name="historieFormeelRelatie" type="ZKN:ZAKBTROVR-basis" minOccurs="0"/>
					<element name="heeftAlsAanspreekpunt" type="ZKN:ZAKBTROVRCTP-basis" nillable="true" minOccurs="0"/>
				</sequence>
				<attribute ref="StUF:entiteittype" fixed="ZAKBTROVR"/>
			</restriction>
		</complexContent>
	</complexType>
```